site/viamguides/src/air-quality/air-quality.md

```diff
@@ -0,0 +1,340 @@
+    ```bash
+    $ mkdir process
+    $ cd process
+    ```
+1. Install the Python package manager.
+    ```bash
+    $ sudo apt install python3-pip
+    ```
+1. Install the Viam Python SDK **into that new directory**.
+    ```bash
+    $ pip3 install --target=process viam-sdk
+    ```
+1. Display the full path of the current directory you are working in on your Raspberry Pi with the `pwd` command. Make a note of this output for the next steps.
+    ```bash
+    $ pwd
+    ```
+1. Find the executable path of Python3 to run `process.py` on your Raspberry Pi with `which Python3`. Again, make a note of this output for the next steps.
+    ```bash
+    $ which Python3
+    ```
+1. Run the following command from your computer (not the SSH prompt to your Raspberry Pi) to copy the code from your computer to your Raspberry Pi. In the command, you will copy `process.py` over to your Raspberry Pi, with the section following the colon `:` indicating where your file should be copied to on the Raspberry Pi (the path of the directory you are working in on your Raspberry Pi, along with the filename). 
+    ```bash
+    $ scp process.py user@host.local:/home/myboard/process/process.py
+    ```
+
+### Configure a Viam process
+
+1. Now let's allow `viam-server` to run the process as the root user on your Raspberry Pi by configuring a [Viam process](https://docs.viam.com/configure/processes/). In [the Viam app](https://app.viam.com/robots) under the **CONFIGURE** tab, click the **+** icon in the left-hand menu and select **Process**. 
+1. Find the corresponding card to `process-1`. Enter the executable path of Python3 running on your Raspberry Pi that you output from a previous step. Add an argument of the `process.py` file to run on your Raspberry Pi. Enter the working directory where you want the process to execute.
+  ![configure process](assets/configureProcess.png)
+1. Still within the `process-1` card, select the advanced settings icon near the top right corner to review the configuration JSON. Create a new `env` property, and add your environment variables within the new property, formatted like the following with your own credentials. 
+    ```json
+      "env": {
+        "SENSOR_NAME": "sensor-1",
+        "PLUG_NAME": "generic-1",
+        "ROBOT_API_KEY": "your-api-key",
+        "ROBOT_API_KEY_ID": "your-api-key-id",
+        "ROBOT_ADDRESS": "your-robot-address"
+      },
+    ```
+    ![configure JSON](assets/configJSON.png)
+    > aside negative
+    > The `SENSOR_NAME` and `PLUG_NAME` are the default names for our air sensor and smart plug when added to our Viam machine. Other machine credentials can be found under the **CONNECT** tab, selecting an SDK, and toggling **Include API key** to reveal your credentials within the code sample.
+        ![get credentials](assets/apiKey.png)
+1. **Save** your updates.
+1. Restart the machine.
```

Comment:
Was it necessary to restart viam-server in order for the process to start? It _should_ happen as soon as the process configuration has been saved. 

Also, the "Restart" button is restarting the server not the device, which might be confusing if we say "restart the machine":

```suggestion
1. Restart viam-server.
```